hal/buildroot/cvm-monitor.sh

```
@@ -0,0 +1,267 @@
+            echo "Cube CVM process:"
+            pgrep -f "$VM_NAME" -l || pgrep -f "/etc/cube/bzImage" -l || echo "Detection method unclear"
+        else
+            echo "✗ Cube CVM is not running"
+        fi
+        
+        echo ""
+        echo "All QEMU processes on system:"
+        pgrep -f "qemu-system-x86_64" -l | head -5 || echo "No QEMU processes found"
+        
+        if [ -f "$PIDFILE" ]; then
+            local pid=$(cat "$PIDFILE" 2>/dev/null)
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)